circuit_knitting/cutting/cutting_experiments.py

```diff
@@ -156,6 +165,25 @@ def generate_cutting_experiments(
                 _append_measurement_circuit(new_qc, cog, inplace=True)
                 subexperiments_dict[label].append(new_qc)
 
+    # Remove initial and final resets from the subexperiments.  This will
+    # enable the `Move` operation to work on backends that don't support
+    # `Reset`, as long as qubits are not re-used.  See
+    # https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/issues/452.
+    # While we are at it, we also consolidate each run of multiple resets
+    # (which can arise when re-using qubits) into a single reset.
+    pass_manager = PassManager()
+    pass_manager.append(
+        [
+            RemoveResetInZeroState(),
+            RemoveFinalReset(),
+            ConsolidateResets(),
+            DAGFixedPoint(),
+        ],
+        do_while=lambda property_set: not property_set["dag_fixed_point"],
```

Comment:
So you repeat these passes until this condition is False. What exactly triggers this? I haven't come up to speed on how control flow operations work yet.

Reply:
The `DAGFixedPoint` pass [sets this property](https://qiskit.org/documentation/stubs/qiskit.transpiler.passes.DAGFixedPoint.html).  It's neat; I learned about it only by reading [the tests](https://github.com/Qiskit/qiskit/blob/e91947e1b55f6e661c683fba72ab66c5f92d3aec/test/python/transpiler/test_remove_reset_in_zero_state.py#L81-L102) for the `RemoveResetInZeroState` pass in qiskit.